README.md

```diff
@@ -14,8 +14,8 @@ Python interfaces to fast, compiled object-oriented algorithms and enabling tech
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/pytrilinos-feedstock-conda-forge-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/pytrilinos-feedstock-conda-forge-feedstock)
```

Comment:
I wonder what caused `pytrilinos-feedstock-conda-forge-feedstock`...

Reply:
That happens when your local git folder's name is different.

Reply:
Thanks. I never saw that :grimacing: